internal/rbd/rbd_util.go

```diff
@@ -114,8 +114,10 @@ type rbdVolume struct {
 	Topology            map[string]string
 	// DataPool is where the data for images in `Pool` are stored, this is used as the `--data-pool`
 	// 	 argument when the pool is created, and is not used anywhere else
```

Comment:
@Yuggupta27 not introduced with this patch, but please feel free to correct the `DataPool` explanation intendation.

Reply:
Sure, I'll add a commit to fix the same

Reply:
Here https://github.com/ceph/ceph-csi/pull/2154